src/CSS/AddItemForm.css

```diff
@@ -1,3 +1,12 @@
+:root {
```

Comment:
CSS variables should not be copied in here. The idea behind variables is to have one place to update but change the colors everywhere they are applied. With this, we have the same variables defined in 2 CSS files.

The CSS variables should be moved to App.css so the shopping view and add item view have access to them.